app/dashboard/views.py

```diff
@@ -1518,7 +1519,21 @@ def change_bounty(request, bounty_id):
         for key in keys:
             value = params.get(key, '')
             old_value = getattr(bounty, key)
+
+            if key == 'bounty_reserved_for_user' and value is not None:
```

Comment:
Value will never be `None` here since it's defaulting to an empty string on L1520